Gemfile

```diff
@@ -16,8 +16,6 @@ gem "react-rails"
 
 gem 'accountingjs-rails'
 gem "jquery-rails"
-gem "turbolinks"
```

Comment:
@betesh I noticed a different javascript error on the site and believe that was probably my issue.  I've reverted the commit that removed turbolinks.

```Started GET "/assets/select2-init.self-19f8bb5e037e797208ca3531f080a4979215bb6f27656265c637dab59be05049.js?body=1" for ::1 at 2016-01-29 08:36:36 -0800
source=rack-timeout id=972c1681abd40406a0ee34284f0db115 timeout=20000ms service=8ms state=completed```